src/Playwright/BrowserContext.cs

```diff
@@ -203,13 +203,14 @@ public Task SetExtraHTTPHeadersAsync(IEnumerable<KeyValuePair<string, string>> h
 
         public async Task<string> StorageStateAsync(string path = null)
         {
+            var options = new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
             string state = JsonSerializer.Serialize(
                 await Channel.GetStorageStateAsync().ConfigureAwait(false),
-                Channel.Connection.GetDefaultJsonSerializerOptions());
+                options);
 
             if (!string.IsNullOrEmpty(path))
             {
-                File.WriteAllText(path, state);
+                await File.WriteAllTextAsync(path, state).ConfigureAwait(false);
```

Comment:
I think WriteAllTextAsync is net 5 only.

Reply:
Yup, figured that!